meilix-default-settings/etc/skel/.dmrc

```diff
@@ -0,0 +1,2 @@
+[Desktop]
```

Comment:
should be part of another pull request

Reply:
Yes, there is a different PR already.
https://github.com/fossasia/meilix/pull/282